rust/arrow/src/datatypes.rs

```diff
@@ -62,7 +64,8 @@ pub enum DataType {
     Float16,
     Float32,
     Float64,
-    Timestamp(TimeUnit),
+    /// A timestamp with an optional timezone
+    Timestamp(TimeUnit, Option<String>),
```

Comment:
Can we change this Option<Arc<String>> so that we can decrease cost of clone operation of DataType?

Reply:
I've changed this, PTAL